apps/users/serializers/admin_account.py

```diff
@@ -19,24 +19,24 @@ class Meta:
             "nickname",
             "name",
             "birthday",
-            "role",
             "status",
-            "joined_at",
-            "withdrawal_requested_at",
+            "role",
+            "withdraw_at",
+            "created_at",
         ]
 
     def get_role(self, obj: User) -> str:
         if obj.is_superuser:
-            return "Admin"
+            return "admin"
         if obj.is_staff:
-            return "Staff"
-        return "User"
+            return "staff"
+        return "user"
 
     def get_status(self, obj: User) -> str:
         status_value: Optional[str] = obj.status_value
 
         if status_value is not None:
             return status_value
         if obj.is_active:
-            return "Active"
-        return "Inactive"
+            return "active"
+        return "inactive"
```

Comment:
```suggestion
    def get_status(self, obj: User) -> str:
        if obj.is_active:
            return "active"
        else:
            if obj.withdrawals.exists():
                return "withdrew"
            else:
                return "inactive"
```
요구사항 대로라면 이렇게 3가지가 반환되어야하지않을까요